### DISCUSSION TITLE
Download Example

### DISCUSSION THREAD

**Discussion by chunmingwang** [XusinboyBekchanov/VisualFBEditor#427] [General]
# Download
Example for VisualFBEditor

[URLDownloadToFile](https://learn.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/platform-apis/ms775123(v=vs.85))

![Download](https://user-images.githubusercontent.com/35757455/199168803-02b1b4b2-7639-4738-895f-fb6dfd13554e.gif)

**Comment by XusinboyBekchanov** [comment]
Good example. Thank you.
